README.md

```diff
@@ -1,9 +1,24 @@
-Dune - A Composable Build System
-================================
-
-[![Main workflow][workflow-badge]][workflow]
-[![Release][release-img]][release]
-[![Coverage Status][coverall-badge]][coverall]
+![Dune][logo]
+
+# A Composable Build System for OCaml
+
+[![Main Workflow][workflow-badge]][workflow]
+[![Release][release-badge]][release]
+[![Coverage][coverage-badge]][coverage]
+[![License][license-badge]][license]
+[![Contributors][contributors-badge]][contributors]
+
+[logo]:               doc/assets/imgs/dune_logo_459x116.png
```

Comment:
Can you un-align these links, and the ones below (by sticking to `]: https` ? As this diff shows, alignment is painful to maintain

Reply:
They were aligned before so I was just sticking to the convention. I can remove all of them.